examples/ensemble/plot_gradient_boosting_quantile.py

```diff
@@ -32,41 +36,87 @@ def f(x):
 y += noise
 y = y.astype(np.float32)
 
+# Split into train, test datasets.
```

Comment:
Maybe we could increase the number of points in X and y to make the metrics a bit more stable and limit overfitting. For instance use  200 or 300 points (before train test split).

Just check that the plot does not look too overcrowded.

Reply:
We can also reduce the learning rate.